src/com/xmslyz/luhn/logic/LuhnChecksumValidation.java

```diff
@@ -0,0 +1,29 @@
+package com.xmslyz.luhn.logic;
+
+public class LuhnChecksumValidation implements Validation {
```

Comment:
to ficzer niezwiązny z podziałem na `logic` i `main` proponuję dodawać osobnym PRem

Reply:
Zbyt pędzisz z dewelopmentem aplikacji. To co powinieneś zrobić:
1/ dokończyć zmiany architekturalne w ramach tego PR i go zmergować
2/ w osobnym PR: napisać testy dla `LuhnChecksumComputing` - to pozwoli domknąć ficzer pt. "obliczanie checksumy algorytmem Luhna"
3/ kolejny PR dodać nowy ficzer walidacji checksumy - wraz z testami do niego